src/Common/Menus/Traits/With_Admin_Bar.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Adds menu item to the WP admin bar.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Common\Menus
+ */
+
+namespace TEC\Common\Menus\Traits;
+
+trait With_Admin_Bar {
```

Comment:
If this trait will be used only in menu, or sub-menu, pages, then I would just include it in the competent `abstract` class.

Reply:
It will - but not all of them.